libs-scala/scala-utils/src/main/scala/com/daml/scalautil/nonempty/NonEmptyReturningOps.scala

```diff
@@ -15,14 +15,14 @@ import NonEmptyCollCompat._
 object NonEmptyReturningOps {
   implicit final class `NE Iterable Ops`[A, CC[_], C](
       private val self: IterableOps[A, CC, C with imm.Iterable[A]]
-  ) {
+  ) extends AnyVal {
```

Comment:
For some reason, I'm under the impression implicit classes don't benefit from extending `AnyVal` anymore. Perhaps this is redundant.